### ISSUE TITLE
DLLs not installed

### ISSUE THREAD

**Issue by svigerske** [ERGO-Code/HiGHS#721]
I build HiGHS v1.2 under MSys2 with
```
cmake .. \
  -G "NMake Makefiles" \
  -DCMAKE_BUILD_TYPE=Release \
  -DCMAKE_INSTALL_PREFIX=/path/to/install \
  -DCMAKE_C_COMPILER=icl -DCMAKE_CXX_COMPILER=icl
nmake
nmake install
```
The last step misses installing the HiGHS DLLs into `$prefix/bin`.

It only installs the `.lib` files into `$prefix/lib` and `highs.exe` into `$prefix/bin` (and some non-binary files).

Also without `-DCMAKE_BUILD_TYPE=Release` I seem to get a debug build, while on other platforms `Release` was the default (which seems correct).

**Comment by lgottwald** [comment]
Could you try to use `-DFAST_BUILD=on`?

**Comment by svigerske** [comment]
If I only add `FAST_BUILD=on`, then I get only static libraries. If I do `-DFAST_BUILD=on -DBUILD_SHARED_LIBS=on`, then I get a DLL and it is installed by `nmake install`.

**Comment by lgottwald** [comment]
Thanks, I see that this will be fixed

**Comment by lgottwald** [comment]
The issue with the debug build I do not understand though. Could you maybe check if the problem occurs with a fresh build directory? And if it does could you check whether it makes a difference to move the if block on line 19 of the top level CMakeLists.txt (`if(NOT CMAKE_BUILD_TYPE)`), which sets the default build type to release, a few lines up before the `project()` call at line 15?

**Comment by svigerske** [comment]
> The issue with the debug build I do not understand though. Could you maybe check if the problem occurs with a fresh build directory? And if it does could you check whether it makes a difference to move the if block on line 19 of the top level CMakeLists.txt (`if(NOT CMAKE_BUILD_TYPE)`), which sets the default build type to release, a few lines up before the `project()` call at line 15?

Yes, moving the `CMAKE_BUILD_TYPE` before `project()` fixes this problem.

**Comment by lgottwald** [comment]
@galabovaa I don't understand the difference between the FAST_BUILD=on/off and installation of the library here. Do you have an idea?

**Comment by mmuetzel** [comment]
Didn't notice this report before. But is likely fixed now that PR #823 is merged.

**Comment by svigerske** [comment]
Yes, seems to work now.
